Overview.src.html

```diff
@@ -622,34 +631,42 @@ <h4 id=terminology-headers>Headers</h4>
    <td>"<code title>document</code>"
    <td>HTML's navigate algorithm.
   <tr>
+   <td>"<code title>document</code>"
+   <td><code title>child-src</code>
+   <td>HTML's <code>&lt;iframe></code> and <code>&lt;frame></code>
+  <tr>
    <td>"<code title>subresource</code>"
    <td><code title>connect-src</code>
    <td><code>navigator.sendBeacon()</code>, <code>EventSource</code>,
```

Comment:
Either "fetch" (and fetch() can use that too) or the empty string, but if we do the empty string perhaps reports need to get their own destination and/or type?